doc/lua_api.txt

```diff
@@ -3032,6 +3032,11 @@ This is basically a reference to a C++ `ServerActiveObject`
         * `"skybox"`: Uses 6 textures, `bgcolor` used
         * `"plain"`: Uses 0 textures, `bgcolor` used
 * `get_sky()`: returns bgcolor, type and a table with the textures
+* `set_clouds({density, color, ambient})`: set cloud parameters
```

Comment:
What does this mean? Is it a table with `density`, `color` and `ambient` keys, or a 3-item array?

Maybe `set_clouds({density=number, color=ColorSpec, ambient=Colorspec})`?